### PR TITLE
feat: monitor router status

### PR DESCRIPTION
**Pull request by wanglonghong**
## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

File: packages/agents/sequencer/src/bindings/server/index.ts [hunk 1]
```diff
@@ -188,6 +194,21 @@ export const bindServer = async (queueName: string, channel: Broker.Channel): Pr
...
 
+  server.post<{ Body: RouterPingRequest }>(
+    "/router-ping",
+    { schema: { body: RouterPingRequestSchema } },
+    async (req, res) => {
+      const { router, timestamp, signed } = req.body;
+      const signerAddress = verifyMessage(`${RouterPingMessage}-${timestamp}`, signed);
+      if (router.toLowerCase() == signerAddress.toLowerCase()) {
+        await cache.routers.setLastActive(router);
+        return res.status(200).send({ message: "OK" });
```
Comment (preethamr) [inline packages/agents/sequencer/src/bindings/server/index.ts:205]: Should we respond to the router with the current data in the cache ?

File: packages/agents/sequencer/src/lib/operations/execute/fastpath.ts [hunk 2]
```diff
@@ -52,6 +52,13 @@ export const storeFastPathData = async (bid: Bid, _requestContext: RequestContex
     });
   }
 
+  // Update the last bid time for a given router.
+  await cache.routers.setLastBidTime(bid.router, {
+    originDomain: transfer.xparams.originDomain,
```
Comment (preethamr) [inline packages/agents/sequencer/src/lib/operations/execute/fastpath.ts:57]: Would also be useful for routers to know when the last successful/accepted bid was.
Wdyt ?
@wanglonghong

File: packages/agents/sequencer/src/bindings/server/index.ts (hunk 1 above)
Comment (wanglonghong) [reply packages/agents/sequencer/src/bindings/server/index.ts:205]: I think yeah. It should be fine even if the cache cleared cause it's being updated in the next cycle.

File: packages/agents/sequencer/src/lib/operations/execute/fastpath.ts (hunk 2 above)
Comment (wanglonghong) [reply packages/agents/sequencer/src/lib/operations/execute/fastpath.ts:57]: That's good suggestion.

**Review by preethamr — APPROVED** [review]

**Pull request merged**